sql/modifications.sql

```diff
@@ -54,6 +54,9 @@ INSERT INTO limit_orders VALUES (random() * 100, 'ORCL', 152, '2011-08-25 11:50:
 -- commands with expressions that cannot be collapsed are unsupported
 INSERT INTO limit_orders VALUES (2036, 'GOOG', 5634, now(), 'buy', random());
 
+-- commands with mutable or volitale functions 
+DELETE FROM limit_orders WHERE id = 246 AND bidder_id = (random() * 1000);
+
```

Comment:
Can you add another test with a function which is stable but not mutable? That way we'll have one test that checks `VOLATILE` and another checking `STABLE`.